### ISSUE TITLE
Organize the project

### ISSUE THREAD

**Issue by travnick** [joboccara/NamedType#37]
@joboccara Thanks for the great article and the implementation!

It would be nice to have all (useful) PR merged, and all issues resolved.
Additionally, I'm about to create a NuGet package, and maybe another for Arch Linux.

To achieve that, some versioning and releases are somehow required.
As far as I can see there is no activity since March 2019, so I suggest that the "main repository" should be moved to the another (more active) fork.

@AMS21 I'm not sure if the https://github.com/AMS21/NamedType/network is the right one.

The whole message is here to organize others contributors, so it will be possible to merge all the valuable work into one repository and release one consistent version.

Best regards!

**Comment by joboccara** [comment]
Yes, I agree! Unfortunately right now I'm struggling to find focus to allocate on this project. If that can be of any comfort, this is in my backlog.

**Comment by joboccara** [comment]
Hi @travnick,
I merged nearly all PRs, so the repository is now up to date!
Should we close this issue?

**Comment by travnick** [comment]
Hi @joboccara,

That's great!

but the versioning and preparing NuGet package is still active topic, so it may be extracted as a new task to do.
